rudder-lang/deny.toml

```diff
@@ -0,0 +1,20 @@
+# Only non-default values are listed here
+
+[licenses]
+# List of explictly allowed licenses
+# See https://spdx.org/licenses/ for list of possible licenses
+# [possible values: any SPDX 3.7 short identifier (+ optional exception)].
+allow = [
+    "MIT",
+    "Apache-2.0",
+    "ISC",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "GPL-3.0",
```

Comment:
what about the or-later, is it handled automatically ?

Reply:
It seems so, yes